.github/ISSUE_TEMPLATE/design.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Design
+about: Track or request design work
+title: 'Design [...] so that [...].'
+labels: design
```

Comment:
This label won't work with our new labels, correct? I'm not sure how we can make it work for two different variants.

Reply:
Oh, good catch! I need to include the emoji.